OpenEXR/exr2aces/main.cpp

```diff
@@ -87,6 +87,9 @@ usageMessage (const char argv0[], bool verbose = false)
 		"\n"
 		"Options:\n"
 		"\n"
+        "-x        support large images: remove 65535 pixel limit on image\n"
+        "          width/height (requires significant memory to process)\n"
+		"\n"        
```

Comment:
Looks like you've got the odd space vs. tab issue. We really need to convert the whole repo to clang-format or equivalent.